sigle/src/modules/home/components/StoryItem.tsx

```diff
@@ -97,81 +105,222 @@ export const StoryItem = ({
+          },
+        }}
+      >
+        {!story.coverImage && story.featured && (
+          <Box
+            css={{
+              p: '$2',
+              backgroundColor: '$gray1',
+              br: '$round',
+              color: '$orange10',
+            }}
+          >
+            <StarFilledIcon />
+          </Box>
+        )}
+        {story.coverImage && (
+          <Link href="/stories/[storyId]" as={`/stories/${story.id}`} passHref>
+            <Box
+              css={{
+                borderRadius: '$1',
+                lineHeight: 0,
+                position: 'relative',
+              }}
+              as="a"
+            >
+              <Box
+                as="span"
+                css={{
+                  display: 'inline-block',
+                  position: 'relative',
+                  overflow: 'hidden',
+                  borderRadius: '$1',
+
+                  '&::before': {
+                    content: '',
+                    position: 'absolute',
+                    left: 0,
+                    top: 0,
+                    width: '100%',
+                    height: '100%',
+                    backgroundColor: '$gray11',
+                    opacity: 0,
+                    transition: '.2s',
+                  },
+
+                  '&:hover::before': {
+                    opacity: 0.1,
+                  },
+                }}
+              >
+                {story.featured && (
+                  <Box
+                    css={{
+                      p: '$2',
+                      backgroundColor: '$gray1',
+                      br: '$round',
+                      color: '$orange10',
+                      position: 'absolute',
+                      top: '-$2',
+                      right: '-$2',
+                    }}
+                  >
+                    <StarFilledIcon />
+                  </Box>
+                )}
+                <StoryImage src={story.coverImage} />
+              </Box>
+            </Box>
+          </Link>
+        )}
+        <Flex direction="column" css={{ flex: 1 }}>
+          <Box
+            css={{
+              '@md': {
+                mb: '$5',
+              },
+            }}
+          >
+            <Flex
+              justify="between"
+              css={{ gap: '$5', mb: '$1', '@md': { mb: '$2' } }}
+            >
               <Link
                 href="/stories/[storyId]"
                 as={`/stories/${story.id}`}
                 passHref
               >
-                <a>{story.title}</a>
+                <a>
+                  <Heading
+                    css={{
+                      fontSize: '$2',
+                      lineHeight: '20.4px',
+                      alignSelf: 'center',
+                      fontWeight: 600,
+                      overflow: 'hidden',
+                      display: '-webkit-box',
+                      '-webkit-line-clamp': 3,
```

Comment:
```suggestion
                      'line-clamp': 3,
```
We should remove the webkit prefix as some browsers might not support it :)

Reply:
just tested and when I remove it, it removes the clamp effect. Do you know if it's possible to have both?

Reply:
hmm not sure but will have a look into it :)